tycho-eclipse-plugin/src/main/java/org/eclipse/tycho/eclipserun/EclipseRunMojo.java

```diff
@@ -446,7 +446,7 @@ public LaunchConfiguration createCommandLine(EquinoxInstallation runtime) throws
 					+ ". Current Java runtime will be used");
 		}
 		cli.setJvmExecutable(executable);
-		cli.setWorkingDirectory(project.getBasedir());
+		cli.setWorkingDirectory(project != null ? project.getBasedir() : new File("").getAbsoluteFile());
```

Comment:
If I remember correctly maven always create an empty project for you when you execute it from within a directory without a pom.

Beside that a temp directory might be better to use?

Reply:
> If I remember correctly maven always create an empty project for you when you execute it from within a directory without a pom.

I don't know that, but I assume the empty project, will then have the current working directory as directory? So basically the same result.

> Beside that a temp directory might be better to use?

The idea was that the current working directory where the direct CLI invocation is launched is probably also interesting was working directory of the eclipse run (just 'forwarding' the shell's workding directory to the Eclipse run).